docs/build.sh

```diff
@@ -102,6 +105,8 @@ PROTO_RST="
   /envoy/service/accesslog/v2/als/envoy/service/accesslog/v2/als.proto.rst
   /envoy/type/percent/envoy/type/percent.proto.rst
   /envoy/type/range/envoy/type/range.proto.rst
+  /envoy/service/auth/v2alpha/external_auth/envoy/service/auth/v2alpha/external_auth.proto.rst
```

Comment:
nit: can you alpha order this. This list is a little out of control and alpha ordering helps a bit.